components/Markdown.tsx

```diff
@@ -1,8 +1,32 @@
-import * as marked from "marked";
-
+import { Marked } from "marked";
+import { markedHighlight } from "marked-highlight";
+import Prism from "prismjs";
+import "prismjs/components/prism-bash.min.js";
+import "prismjs/components/prism-javascript.min.js";
+import "prismjs/components/prism-typescript.min.js";
+import "prismjs/components/prism-css.min.js";
+import "prismjs/components/prism-json.min.js";
+import "prismjs/components/prism-jsx.min.js";
+import "prismjs/components/prism-tsx.min.js";
+import "prismjs/components/prism-rust.min.js";
 import { css } from "utils/css.ts";
 import clsx from "clsx";
 
+const languageAliases: Record<string, string> = {
+  rs: "rust",
+  console: "sh",
+};
+
+const marked = new Marked(markedHighlight({
+  emptyLangClass: "language-none",
+  langPrefix: "language-",
+  highlight(code, lang, _info) {
+    lang = languageAliases[lang] ?? lang;
+    const language = Prism.languages[lang] ?? Prism.languages.autoit;
```

Comment:
**question:** Isn't `autoit` the hotkey language? You should default to no highlighting.

Reply:
isnt our default language the autohotkey language?

Reply:
nah wait wrong language its this one https://www.autoitscript.com/site/autoit/